custom_components/alexa_media/helpers.py

```
@@ -95,43 +95,62 @@ async def add_devices(
-            return name
+        if base:
+            return str(base)
 
-        # Only attempt switch reconstruction if attributes were explicitly defined
-        # (avoids MagicMock auto-attribute trap in tests)
-        dev_dict = getattr(dev, "__dict__", {})
+        return None
 
-        client = dev_dict.get("_client")
-        suffix = dev_dict.get("_unique_id_suffix")
+    def _device_base_name(dev: Entity) -> str | None:
```

Pylint (4.0.5)

[error] function already defined line 98

(E0102)